site/docs/operations/docker.md

```diff
@@ -0,0 +1,340 @@
+command without starting a full instance:
+
+```bash
+docker run hazelcast/hazelcast-jet jet --version
+```
+
+You should get output similar to:
+
+```text
+$ docker run hazelcast/hazelcast-jet jet --version
+Hazelcast Jet {jet-version}
+Revision e6c60a1
+Build 20200429
+```
+
+If your `hazelcast/hazelcast-jet` latest image points to an older
+version you can update it by running the following command:
+
+```bash
+docker pull hazelcast/hazelcast-jet
+```
+
+You can also specify a particular version of the image to run when
+starting the container, e.g. when you need to run an older version of
+Jet:
+
+```bash
+docker run hazelcast/hazelcast-jet:4.0
+```
+
+## Submit job to Hazelcast Jet
+
+We will submit a hello world job, which is available in the Hazelcast
+Jet distribution. Download the Jet distribution from
+[here](https://github.com/hazelcast/hazelcast-jet/releases/download/v{jet-version}/hazelcast-jet-{jet-version}.tar.gz)
+. From now and on we will refer to the extracted location as `<jet home>`.
+
+Also, note the address of your Hazelcast Jet instance from the log:
+
+```text
+Members {size:1, ver:1} [
+    Member [172.17.0.2]:5701 - c54413e4-2cd2-4c07-91ec-d5992b18f9b0 this
+]
+```
+
+Run the following command to submit the hello world job to your Hazelcast
+Jet instance:
+
+```bash
+docker run -it -v "$(pwd)"/examples:/examples hazelcast/hazelcast-jet jet -a 172.17.0.2 submit /examples/hello-world.jar
+```
+
+The parameters are as follows:
+
+* -it allocates interactive terminal, allowing you to cancel the submit
+  command by pressing Ctrl+C
+* -v "$(pwd)"/examples:/examples mounts folder `examples` from your
+  current directory to `/examples` directory inside the container
+* -a 172.17.0.2 - shortcut for `--addresses`, specifies an address of
+  running Hazelcast Jet instance, needs to be specified because the
+  instance is not running on localhost:5701
+* `submit` command to submit a job
+* `/examples/hello-world.jar` path to a jar with a job inside the
+  container
+
+You can list the running jobs inside the cluster by running the
+following command:
+
+```bash
+docker run hazelcast/hazelcast-jet jet -a 172.17.0.2 list-jobs
+```
+
+which gives an output similar to:
+
+```text
+$ docker run hazelcast/hazelcast-jet jet -a 172.17.0.2 list-jobs
+ID                  STATUS             SUBMISSION TIME         NAME
+045e-987c-1940-0001 RUNNING            2020-05-18T20:08:03.020 hello-world
+```
+
+You can cancel a running job by running the following command:
+
+```bash
+docker run hazelcast/hazelcast-jet jet -a 172.17.0.2 cancel hello-world
+```
+
+### Using local distribution
+
+Alternatively you can use the downloaded distribution to submit a job to
+a cluster directly, from `<jet home>` run:
+
+```bash
+bin/jet submit examples/hello-world.jar
+```
+
+In some environments (e.g. on Windows or macOS) the container is not
+directly accessible via network from your host machine. When starting
```

Comment:
I think you should enable [Host Networking](https://docs.docker.com/network/host/) to make this happen OOTB. But we shouldn't go that way since it has it's own drawbacks. It's better to always make port mapping for consistency.

Reply:
The page you link says Host networking is not available on Mac/Windows.